guacamole/src/main/frontend/src/translations/en.json

```diff
@@ -207,17 +207,21 @@
         "ERROR_EMPTY_FILE": "The provided file is empty",
         "ERROR_INVALID_CSV_HEADER": "Invalid CSV Header \"{HEADER}\" is neither an attribute or parameter",
         "ERROR_INVALID_MIME_TYPE": "Unsupported file type: \"{TYPE}\"",
+        "ERROR_DETECTED_INVALID_TYPE": "Unsupported file type. Please make sure the file is valid CSV, JSON, or YAML.",
```

Comment:
Since you're fixing issues - maybe the style in this section could be cleaned up to match the style used throughout the rest of the translation files - mainly that colons are generally aligned with the items above and below? Unless we're working to change that style and go with something more like what's in this section across the board? I'm good either way, just going for consistency... :smile:

Reply:
Oh, good call! It's done.